conf.pri.mingw

```diff
@@ -0,0 +1,58 @@
+# Adapt these paths
+
+# Point this to the boost include folder
+INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_51_0)
+# Point this to the libtorrent include folder
+INCLUDEPATH += $$quote(C:/qBittorrent/RC_0_16/include)
+# Point this to the zlib include folder
+INCLUDEPATH += $$quote(C:/qBittorrent/Zlib/include)
+# Point this to the openssl include folder
+INCLUDEPATH += $$quote(C:/qBittorrent/openssl/include)
+
+# Point this to the boost lib folder
+LIBS += $$quote(-LC:/qBittorrent/boost_1_51_0/stage/lib)
+# Point this to the libtorrent lib folder
+LIBS += $$quote(-LC:/qBittorrent/RC_0_16/bin/path-according-to-the-build-options-chosen)
+# Point this to the zlib lib folder
+LIBS += $$quote(-LC:/qBittorrent/Zlib/lib)
+# Point this to the openssl lib folder
+LIBS += $$quote(-LC:/qBittorrent/openssl/lib)
+
+# Adapt the lib names/versions accordingly
+CONFIG(debug, debug|release) {
+  LIBS += libtorrent-rasterbar \
+          libboost_system-mt \
+          libboost_filesystem-mt \
+          libboost_thread_win32-mt
+} else {
+  LIBS += libtorrent-rasterbar \
+          libboost_system-mt \
+          libboost_filesystem-mt \
+          libboost_thread_win32-mt
+}
+
+LIBS += libadvapi32 libshell32 libuser32 libole32
+LIBS += libcrypto libssl libwsock32 libws2_32 libz libiconv
+LIBS += libpowrprof
+
+DEFINES += NTDDI_VERSION=0x05010000
+DEFINES += _WIN32_WINNT=0x0501
+DEFINES += _WIN32_IE=0x0501
+# Boost 1.60+ defaults to Vista+ support. The define below enables XP support again.
```

Comment:
Maybe we can again discuss dropping the XP support?

Reply:
for me, **iff** it doesn't trade off something & it's easy to support it, I won't bother.

Reply:
IIRC, currently build-time support mostly needed. So it's @sledgehammer999 headache... Although, as I recall, we still have some code associated with the XP support. In any case, I'm not against it as long as it does not impede the further development of the project.